### ISSUE TITLE
wrong minification of CSS comments

### ISSUE THREAD

**Issue by locnide** [magiclen/html-minifier#9]
For some situations where I have comments in `<style>` tags, the minified CSS is wrong and doesn't correspond to the original css.
 
As an example, minifying
```html
<!DOCTYPE html>
<html>
    <head>
        <style>
            .test1 {
                font-weight: 30em;
            }/**/
            .test2 {
            }/**/
            .test3 {
            }/**/
        </style>
    </head>
    <body>
    </body>
</html>
```
gives me 

```html
<!DOCTYPE html>
<html>
<head>
<style>.test1{font-weight:30em;}/*!/
            .test2 {
            }/**/
.test3{}
        </style>
</head>
<body>
</body>
</html>
```
(notice the `/*!/` that makes `.test2` part of the comment)

**Comment by locnide** [comment]
It seems that it is due to the CSS minifying library used.
I opened an issue in the `minifier-rs` repository : https://github.com/GuillaumeGomez/minifier-rs/issues/69

**edit**: it have been solved in minifier-rs 0.0.40

**Comment by magiclen** [comment]
A very quick fix, nice!!
